pipeline/splicing/calcVarPriors.py

```diff
@@ -595,27 +598,43 @@ def getVarLocation(variant, boundaries):
             return "UTR_variant"
         return "intron_variant"
 
-def getFastaSeq(chrom, rangeStart, rangeStop):
+def getFastaSeq(chrom, rangeStart, rangeStop, plusSeq=True):
     '''
     Given chromosome (in format 'chr13'), region genomic start position, and
     region genomic end position:
     Returns a string containing the sequence inclusive of rangeStart and rangeStop
-    To get + strand sequence, rangeStart < rangeStop
+    If plusSeq=True, returns plus strand sequence
+    If plusSeq=False, returns minus strand sequence
     '''
-    url = "http://togows.org/api/ucsc/hg38/%s:%d-%d.fasta" % (chrom, rangeStart, rangeStop)
+    if rangeStart < rangeStop:
```

Comment:
The request only works if regionStart < regionEnd (otherwise it returns an empty list). I could potentially change it to `if varStrand == "-"` but I wanted it be a little more flexible. `plusStrandSeq` refers to the sequence that is returned because you can return both plus and minus strand sequence for a specific gene.